pkg/apis/controlplane/helper.go

```diff
@@ -22,3 +22,19 @@ func (r *NetworkPolicyReference) ToString() string {
 	}
 	return fmt.Sprintf("%s:%s/%s", r.Type, r.Namespace, r.Name)
 }
+
+func (r *GroupReference) ToString() string {
+	if r.Namespace == "" {
+		return r.Name
+	}
+	return fmt.Sprintf("%s/%s", r.Namespace, r.Name)
+}
+
+// ToTypedString returns the Group or ClusterGroup namespaced name as a string along with its type.
+// Typical usage of typed string will be to use in log messages.
+func (r *GroupReference) ToTypedString() string {
```

Comment:
There is a naming inconsistency with the above function. Do we need the 2 versions or should `ToString` just include the type as well?

Reply:
The `ToString` was actually used in place of `r.Name` for most usages, so I changed the func name. `ToTypedString` has more info for log messages.